rpcs3/Emu/RSX/GL/GLPipelineCompiler.cpp

```diff
@@ -38,6 +38,10 @@ namespace gl
 
 	void pipe_compiler::operator()()
 	{
+#ifndef _WIN32
+		// Set low priority
+		thread_ctrl::scoped_priority low_prio(-1);

```

Comment:
Handle the platform exclusivity in the implementation. That is an implementation detail and has nothing to do with the emulator if a platform doesn't allow priority lowering.